Packs/SophosCentral/Integrations/SophosCentral/SophosCentral.py

```diff
@@ -0,0 +1,1764 @@
+                                      headers=table_headers, removeNull=True)
+    return CommandResults(outputs_key_field='id', outputs_prefix='SophosCentral.ExploitMitigation',
+                          raw_response=result, outputs=object_data, readable_output=readable_output)
+
+
+def sophos_central_exploit_mitigation_delete_command(client: Client, args: dict) -> CommandResults:
+    """
+    Delete an existing exploit mitigation.
+
+    Args:
+        client (Client): Sophos Central API client.
+        args (dict): All command arguments
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    result = client.delete_exploit_mitigation(args.get('mitigation_id', ''))
+    readable_output = f'Success deleting exploit mitigation: {args.get("mitigation_id", "")}'
+    outputs = {'deletedMitigationId': args.get('mitigation_id', '')}
+    if not result or not result.get('deleted'):
+        readable_output = f'Failed deleting exploit mitigation: {args.get("mitigation_id", "")}'
+        outputs = {}
+    return CommandResults(raw_response=result, readable_output=readable_output, outputs=outputs,
+                          outputs_prefix='SophosCentral.DeletedExploitMitigation')
+
+
+def create_detected_exploit_output(item: Dict) -> Dict:
+    """
+    Create the complete output dictionary for a detected exploit.
+
+    Args:
+        item (dict(str)): A source dictionary from the API response.
+
+    Returns:
+        object_data (dict(str)): The output dictionary.
+    """
+    data_fields = ['id', 'thumbprint', 'count', 'description', 'firstSeenAt', 'lastSeenAt']
+    outputs = {field: item.get('field') for field in data_fields}
+
+    last_user = item.get('lastUser')
+    if last_user:
+        outputs['lastUserName'] = last_user.get('name')
+        outputs['lastUserId'] = last_user.get('id')
+
+    last_endpoint = item.get('lastEndpoint')
+    if last_endpoint:
+        outputs['lastEndpointHostname'] = last_endpoint.get('hostname')
+        outputs['lastEndpointId'] = last_endpoint.get('id')
+
+    return outputs
+
+
+def sophos_central_detected_exploit_list_command(client: Client, args: dict) -> CommandResults:
+    """
+    List all detected exploits.
+
+    Args:
+        client (Client): Sophos Central API client.
+        args (dict): All command arguments
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    results = client.list_detected_exploit(min(int(args.get('page_size', '')), 100),
+                                           args.get('page'), args.get('thumbprint_not_in'))
+    items = results.get('items')
+    table_headers = ['id', 'description', 'thumbprint', 'count', 'firstSeenAt', 'lastSeenAt']
+    outputs = []
+    if items:
+        for item in items:
+            current_object_data = create_detected_exploit_output(item)
+            outputs.append(current_object_data)
```

Comment:
if no items?
handle the HR accordingly.
please try it and make sure that the API does not return an error, if he does - catch it to print a normal war room entry.

Reply:
Not a problem on lists